_posts/0000-01-14-louievandyke.md

```diff
@@ -6,4 +6,4 @@ title: "Welcome to our slide deck!"
 ![puss-in-boots](https://cloud.githubusercontent.com/assets/16547949/25401020/740dd682-29c2-11e7-9906-ac5c3a8496c7.jpg)
 {: .center}
 
-Caption here
+I love training on Mondays...
```

Comment:
Maybe be more specific about what kind of training, like `I :heart: GitHub training on Mondays`.

Reply:
comments from Aninda